bot/exts/valentines/be_my_valentine.py

```diff
@@ -88,26 +87,18 @@ async def send_valentine(
             msg = "You are supposed to use this command in the server."
             return await ctx.send(msg)
 
-        if user:
-            if Lovefest.role_id not in [role.id for role in user.roles]:
-                message = f"You cannot send a valentine to {user} as he/she does not have the lovefest role!"
-                return await ctx.send(message)
+        if Lovefest.role_id not in [role.id for role in user.roles]:
+            message = f"You cannot send a valentine to {user} as he/she does not have the lovefest role!"
+            return await ctx.send(message)
```

Comment:
I know this is what it was previously, but I'm not sure why we are returning the message that was sent. I think this should actually be
```
await ctx.send(message)
return
```